components/analytics/clickhouse/README.md

```diff
@@ -0,0 +1,38 @@
+<!--
+ Copyright (C) CVAT.ai Corporation
+
+ SPDX-License-Identifier: MIT
+-->
+
+# Clickhouse DB migrations
+
+This directory contains migrations for the Clickhouse DB used by CVAT.
+
+## Implementation details
+
+The directory contains initialization scripts for the Clickhouse instance.
+Only the main script `init.sh` is an executable shell script. The migration files
+are also shell scripts inside, but they have the `.cmf` extension, which stands
+for "clickhouse migration file". We use a custom extension to avoid automatic execution
+of these files by Clickhouse.
```

Comment:
It seems that this reasoning doesn't hold if we no longer mount this directory into the server container. So we could rename the files to `*.sh`, to make them easier to work with in IDEs.